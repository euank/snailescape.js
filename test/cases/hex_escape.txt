\x21\x08\x7\x20\x20\x31\x32\x7D\x7dddd\x7toohigh
