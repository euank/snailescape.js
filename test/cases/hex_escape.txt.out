["!\b\u0007  12}}ddd\u0007toohigh"]
